metrics/IdentityManagement/PasswordRotationFrequency/metric.rego

```diff
@@ -0,0 +1,18 @@
+package cch.metrics.password_rotation_frequency
+
+import data.cch.compare
+import rego.v1
+
+default applicable = false
+
+default compliant = false
+
+freq := input.policyDocument.passwordRotationFrequency
+
+applicable if {
+	freq != null
```

Comment:
We can use just "freq" to check if this field exists. Checking for not being null only helps if we have cases where the field exists but the value is null. But for these cases, I would also say, that the metric is applicable but the Eivdence is not filled properly and therefore evalutes to non-compliant.